packages/expo-manifests/ios/EXManifests/Manifest.swift

```diff
@@ -322,6 +322,56 @@ public class Manifest: NSObject {
     return jsEngine
   }
 
+  /**
+   Queries the dedicated package properties in `plugins`
+   */
+  public func getPluginProperties(packageName: String) -> [String: Any]? {
+    typealias PluginWithProps = (String, [String: Any])
+    typealias PluginWithoutProps = String
+    enum PluginType {
+      case withProps (PluginWithProps)
+      case withoutProps (PluginWithoutProps)
+
+      private static func fromRawValue(_ optionalValue: Any?) -> PluginType? {
+        guard let value = optionalValue else {
+          return nil
+        }
+        if let valueArray = value as? [Any],
+          let name = valueArray[0] as? String, let props = valueArray[1] as? [String: Any] {
+          return .withProps((name, props))
+        }
+        if let value = value as? String {
+          return .withoutProps(value)
+        }
+        let exception = NSException(
+          name: NSExceptionName.internalInconsistencyException,
+          reason: "Value for (key = plugins) has incorrect type",
+          userInfo: ["key": "plugins"]
+        )
+        exception.raise()
+        return nil
+      }
+
+      static func fromRawArrayValue(_ value: [Any]) -> [PluginType]? {
+        return value.compactMap { fromRawValue($0) }
+      }
+    }
+
+    guard let pluginsRawValue = expoClientConfigRootObject()?["plugins"] as? [Any],
```

Comment:
nit: maybe prefer `optionalValue(forKey: "plugins")` to match other methods in this class, unless you want incorrect types to also be ignored (as written incorrect types will be skipped)